docs/compose/Skeleton/README.md

```diff
@@ -0,0 +1,70 @@
+# Skeleton
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
```

Comment:
a minor documentation improvement: it'd be great if you could put a short summary at the top to highlight that generally you need to use the skeleton within a shimmer overlay for the right effect. maybe move the shimmer overlay to the top and describe it there?

Reply:
ok